arches_component_lab/src/arches_component_lab/generic/GenericFormField.vue

```diff
@@ -20,31 +23,44 @@ const emit = defineEmits<{
 
 const formFieldRef = useTemplateRef("formField");
 
+const derivedInitialValue = computed(() => {
+    const defaultSlotNodes = slots.default?.();
+    const firstVNodeProps = defaultSlotNodes?.[0].props as Record<
+        string,
+        unknown
+    >;
+
+    if (!firstVNodeProps) {
+        return null;
+    }
+
+    return firstVNodeProps["value"] || firstVNodeProps["model-value"];
```

Comment:
If a project uses camel case to pass props to child components, this won't match `modelValue`. `modelValue` happens to be how it's doc'd in the primevue docs. I like that we're following the kebab case convention, and I get that we expect projects to follow our style, but is it worth being slightly more lenient here and just accepting `modelValue` also?